Tasks/Alexeev/task1/bank.py

```diff
@@ -0,0 +1,6 @@
+import math
+P = 20000
+N = 0.15 
+t = 5*12
+S = P * (1+(N/12)) ** t 
+print("Cash = " + str(S)) 
```

Comment:
Cash = 42143.62693902471 - if only it has no decimal point)) Please do not scare a user with such huge values, round them instead.